docs/README.md

```diff
@@ -0,0 +1,15 @@
+# Epic EI Connector
+The Epic [connector](https://docs.wso2.com/display/EI611/Working+with+Connectors) allows you to access the Epic FHIR APIs through the WSO2 EI. 
+Epic connector currently supports the DSTU 2 Final (1.0.2) version of the FHIR standard.
```

Comment:
Mention which authorization mechanism you are using and provide the api doc as well